contracts/libraries/Assessment/AssessmentIncidentsLib.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+
+import "../../interfaces/INXMToken.sol";
+import "../../interfaces/IMemberRoles.sol";
+import "../../interfaces/IPool.sol";
+import "../../interfaces/ICover.sol";
+import "../../interfaces/IAssessment.sol";
+import "../../interfaces/IMasterAwareV2.sol";
+import "../../libraries/Assessment/AssessmentVoteLib.sol";
+
+library AssessmentIncidentsLib {
```

Comment:
why not just IncidentsLib?

Reply:
Do we have any guarantees that we won't have clashes with libraries from modules that have similar functionality? (i.e. 1 xLib in assessment and another xLib in cover)